docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -252,6 +252,9 @@ The following example formats assorted floating-point values with the number for
 [!code-csharp[Formatting.Numeric.Standard#6](../../../samples/snippets/csharp/VS_Snippets_CLR/Formatting.Numeric.Standard/cs/Standard.cs#6)]
 [!code-vb[Formatting.Numeric.Standard#6](../../../samples/snippets/visualbasic/VS_Snippets_CLR/Formatting.Numeric.Standard/vb/Standard.vb#6)]
 
+> [!IMPORTANT]
+> The format for a specific culture may change over time. Operating system settings, user settings, environment variables, and changes to .NET itself, may affect the format of a specific culture. Starting with .NET 5.0, .NET tries to unify cultural formats across platforms. For more information, see [.NET globalization and ICU](../globalization-localization/globalization-icu.md).
```

Comment:
```suggestion
> The format you see for a specific culture might differ from the following examples. Operating system settings, user settings, environment variables, and the .NET version you're using can all affect the format. For example, starting with .NET 5, .NET tries to unify cultural formats across platforms. For more information, see [.NET globalization and ICU](../globalization-localization/globalization-icu.md).
```

Reply:
What about adding **result**, as it's not the format that changes, but the result.

```suggestion
> The result of a formatted string for a specific culture might differ from the following examples. Operating system settings, user settings, environment variables, and the .NET version you're using can all affect the format. For example, starting with .NET 5, .NET tries to unify cultural formats across platforms. For more information, see [.NET globalization and ICU](../globalization-localization/globalization-icu.md).
```

Reply:
Maybe just "The formatted string for a specific culture...".

Reply:
I still feels like that implies the format string (the "N" you pass to .ToString) is what could change, but it's really the output. OK Thanks for the feedback, I'll fix it up 😄